metadata/containers.go

```diff
@@ -225,7 +225,7 @@ func (s *containerStore) Delete(ctx context.Context, id string) error {
 
 	bkt := getContainersBucket(s.tx, namespace)
 	if bkt == nil {
-		return errors.Wrapf(errdefs.ErrNotFound, "cannot delete container %v, bucket not present", id)
+		return errors.Wrapf(errdefs.ErrNotFound, "cannot delete container %q, bucket %q:%q", id, namespace, id)
```

Comment:
1. duplicated "id"
2. namespace and bucket is unnecessary

Reply:
leaving in namespace since it's helpful for the user